src/NServiceBus.Core/Extensibility/ExtendableOptions.cs

```diff
@@ -5,6 +5,6 @@ namespace NServiceBus.Extensibility
     /// </summary>
     public abstract class ExtendableOptions
```

Comment:
the doco for this seems to imply it is an extension point but everything is internal